roles/container-engine/cri-o/templates/crio.conf.j2

```diff
@@ -377,7 +377,7 @@ enable_metrics = {{ crio_enable_metrics | bool | lower }}
 # The port on which the metrics server will listen.
 metrics_port = {{ crio_metrics_port }}
 
-{% if nri_enabled and crio_version >= v1.26.0 %}
+{% if nri_enabled and crio_version is version('v1.26.0', operator='ge') %}
```

Comment:
Both are correct.
As you wish.

Reply:
HI @ledroide 

Thanks for the PR

In the other code of kubespray , like : https://github.com/kubernetes-sigs/kubespray/blob/master/roles/network_plugin/flannel/tasks/main.yml#L5
Using `>=` is more common than `ge`, so it can make the project more readable for most developers.

And we also need to check the CLA :-)

Reply:
Yup sorry I said simplier but the main reason is to be align with the codebase that mainly use ">=" 😄